rover/source/docs/sailboat-hardware.rst

```diff
@@ -19,10 +17,62 @@ to the wind speed the more important this is. See :ref:`wind-vane`.
+to the boat centre line. 
+
+The sail angle :ref:`SAIL_ANGLE_MIN<SAIL_ANGLE_MIN>` and :ref:`SAIL_ANGLE_MAX<SAIL_ANGLE_MAX>` parameters should be set
+to the angle to boom is to the centre line at each extreme of its travel. This allows the angle of the
+boom to be calculated at any point between. 
+
+The ideal sail angle to the wind :ref:`(SAIL_ANGLE_IDEAL)<SAIL_ANGLE_IDEAL>` should then be set. This defines the angle
+between the boom and the wind direction as reported by the wind vane. An angle of zero here would result
+in the boom staying parallel to the wind vane. The boom will keep this angle to the wind until it reaches
+either its minimum or maximum limit. If the sails are too loose this number should be increased. This can
+be tested in ACRO mode. 
+
+Wing sail elevator
+------------------
+A wing sail may be actuated by a secondary control surface akin to a plane's elevator. This should be setup such that the servo mid point (``SERVOx_TRIM``) results in zero angle of attach on the main wing.
+See `Karl's build <https://discuss.ardupilot.org/t/sailboat-support/32060/321>`__ . Use servo output function, ``SERVOx_FUNCTION``, 128 - Wing Sail Elevator.
+
+The servo min (``SERVOx_MIN``) and max (``SERVOx_MAX``) end points should be set such that the main wing is held in a maximum lift angle of attack on both tacks.
+If the wing sail is acting on the wrong tack the servo should be reversed with ``SERVOx_REVERSE``. The `SAIL_ANGLE_*` parameters are not use with wing sail elevators.
+
+Direct wing sail actuation
+--------------------------
+Wing sail is directly driven by a servo, in this case use servo output function, ``SERVOx_FUNCTION``, 137 - Sail Mast Rotation. See `Ari's build <https://github.com/ArduPilot/ardupilot/pull/17066#issuecomment-813692335>`__ 
+
+The servo mid point (``SERVOx_TRIM``) must be set such such that the wing is held on the center line of the boat. The min (``SERVOx_MIN``) and max points (``SERVOx_MAX``) should be set
+such that the wing moves to the same angle on each tack. This maximum angle should be measured and set to the :ref:`SAIL_ANGLE_MAX<SAIL_ANGLE_MAX>` parameter. As with sheeted sails :ref:`SAIL_ANGLE_IDEAL<SAIL_ANGLE_IDEAL>`
+controls the angle the wing is sheeted to the wind. There is not such visual feedback for under and over sheeting as there is with a soft sail so some testing might be required to find the best value.
+
+Note that to stop the wing sail will be sheeted to the center line of the boat rather than actively driven to point into the wind.
+
+
+Motoring
+---------
+Sailboat also supports motoring, this can be user selected or based on a wind speed threshold. The motor can take any form and should be setup as it would be for a normal rover.
+In this case the throttle stick will control both the motor and the sail. To avoid this conflict sail control can be set to a second RC input channel by setting a `RCx_OPTION` to 207 - Mainsail.
```

Comment:
I would pull this note out of motoring and put it at the top of the page. Even without a motor, the throttle stick is not the same as sheet. It took me a log time to figure out their relationship. For instance, the *throttle* stick must be at min to arm, but sail control can be anywhere. Throttle stick is super idiomatic with sheets--pull to sheet in, push to sheet out (I use that setup on my DF65s). It is less useful for other configurations. On my mast-rotation boat, I have a rotating knob set up as sail control. Rotating it corresponds to the sail rotation direction and magnitude. This works the same in manual and auto modes. I have throttle stick as throttle and leave it at MIN to arm.